src/tests/SelectedRevisionsTable.test.tsx

```diff
@@ -16,16 +18,18 @@ describe('Search View', () => {
 
   it('should delete revisions after click and not show revisions table if no revisions', async () => {
     const dispatchSpy = jest.spyOn(store, 'dispatch');
+    const user = userEvent.setup({ delay: null });
 
-    const { rerender } = render(<SelectedRevisionsTable />);
+    // const { rerender } = render(<SelectedRevisionsTable />);
+    render(<SelectedRevisionsTable />);
     const button = document.querySelectorAll('#close-button');
 
-    await (button[0] as HTMLButtonElement).click();
-    await (button[1] as HTMLButtonElement).click();
```

Comment:
received this error: `Unexpected `await` of a non-Promise (non-"Thenable") value  @typescript-eslint/await-thenable` so I replaced it with userEvent